client/maps.js

```diff
@@ -268,6 +268,121 @@ class MapLoader {
             callback(workspace);
         });
     }
+    
+    getUniques(_callback){
+        let map = {};
```

Comment:
This variable does not need to be scoped outside the inner callback. Callbacks get less efficient the more variables they can "see",

Reply:
(Fix: delete this variable, just do 'let uniques = result || {}' inside the callback from the get.